.gitignore

```diff
@@ -35,4 +35,3 @@ test-results
 .cache/
 .eslintcache
 playwright.env
-firefox
```

Comment:
Maybe there was a reason why this was added in the past? There is no such thing for chromium so I assume it can be removed.

Reply:
The reason it was added is that we download firefox binaries for bidi tests into this directory and we don't want it to appear in `git status` output and I'd still keep it in `.gitignore`. We use bundled build for chromium, so there is no such folder for it.

Reply:
The follow-up discussion can be found at https://github.com/microsoft/playwright/pull/34461#discussion_r1928216866.